Tests/StreamChatTests/APIClient/Endpoints/ChannelEndpoints_Tests.swift

```diff
@@ -212,6 +212,35 @@ final class ChannelEndpoints_Tests: XCTestCase {
             XCTAssertEqual(mute ? "moderation/mute/channel" : "moderation/unmute/channel", endpoint.path.value)
         }
     }
+    
+    func test_muteChannelWithExpiration_buildsCorrectly() {
```

Comment:
Personally, i prefer to have seperate test methods for each case. In that way test results are clear when they are failing. 

One way you can approach that is to create a private assert method and pass the expected result. Based on the result you passed in the assert will verify true or false. Then you can have two test cases one for failing one for succeeding

Reply:
Tbh I just copy-pasted `test_muteChannel_buildsCorrectly` to `test_muteChannelWithExpiration_buildsCorrectly` and added an expiration. Not sure it makes sense to rewrite both of them, also considering this feature will be replaced by OpenAPI, but let me know what you think.

Reply:
Definitely not a reason there. Thanks for clarifying it :)